database/USAFacts_collector

```diff
@@ -0,0 +1,105 @@
+############################################################################################################################
+################################## Pulling and Transforming USAFacts County Level Data ##################################### 
+############################################################################################################################
+# Script for taking USAFacts county-level dataset, adding lat/long coordinates for each county, and transforming datasheet
+# into NECSI time-series format
+# Only one required package:
+library(readr)
+
+#### Set working directory ####
+setwd("~/Documents/Data_and_Projects/NECSI-COVID19/Data/USA_CountyLevel")
```

Comment:
This is not portable!

Reply:
Understood, should I just comment to let users know that they need to specify a path to a directory that they would like to write output from if they wish to do so?

Reply:
You should use a relative path to somewhere in the repository

Reply:
Should a sub-directory be created in the covid19 directory for this script then so that the script isn't just inhabiting and writing output to covid19?

Reply:
sure, that could work.

Reply:
Should a sub-directory be created in the covid19 directory for this script then so that the script isn't just inhabiting and writing output to covid19? Sorry, never written R code to use github repos as directories.